From 2a77221d2f867d15d283f93267ec130eaf78ace4 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sat, 23 Sep 2017 18:29:21 +0100
Subject: [PATCH 28/39] camera: include: Don't override possible overlayed
 header

 * If targets define a specific header path, then let it be the case
   and don't override it to avoid mismatched headers in the modules.

 * This only became an issue since O due to moved camera headers (commit e2b43843fd12783188edd2c54188ea8d26864788).

Change-Id: I6bf7d1da7448d71f7821d40501db971f837781db

Signed-off-by: Dominggoes Isakh <drjisakh@gmail.com>
---
 camera/include/camera/CameraParameters2.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/camera/include/camera/CameraParameters2.h b/camera/include/camera/CameraParameters2.h
index f691cd6..5fae079 100644
--- a/camera/include/camera/CameraParameters2.h
+++ b/camera/include/camera/CameraParameters2.h
@@ -19,7 +19,7 @@
 
 #include <utils/Vector.h>
 #include <utils/String8.h>
-#include "CameraParameters.h"
+#include <camera/CameraParameters.h>
 
 namespace android {
 
-- 
2.7.4

